Source/hydro/trace_ppm.cpp

```diff
@@ -229,19 +229,37 @@ Castro::trace_ppm(const Box& bx,
 
     load_stencil(q_arr, idir, i, j, k, QPRES, s);
 
-    if (use_pslope) {
+    bool in_hse{};
+
+    // HSE pressure on interfaces -- needed if we are dealing with
+    // perturbation pressure as the parabolic reconstruction
+
+    amrex::Real p_m_hse{};
+    amrex::Real p_p_hse{};
+
+    if (castro::use_pslope || castro::ppm_well_balanced) {
         Real trho[nslp];
         Real src[nslp];
 
         load_stencil(q_arr, idir, i, j, k, QRHO, trho);
         load_stencil(srcQ, idir, i, j, k, QUN, src);
 
-        ppm_reconstruct_pslope(trho, s, src, flat, dL, sm, sp);
+        in_hse = ppm_reconstruct_pslope(trho, s, src, flat, dx[idir], sm, sp);
```

Comment:
probably better to pass in `dL` rather than `dx[idir]` although it only makes a difference in spherical-theta direction but gravity is in r so there will be no difference, but I still feel its more consistent this way. Same goes with using `dL` to get `p_m_hse` and `p_p_hse` later.

Reply:
done